playwright/tests/page-objects/editor/search-palette.ts

```diff
@@ -9,8 +10,8 @@ export class SearchPalette {
     this.searchField = this.page.locator('div.autocomplete-palette').locator('input');
   }
 
-  async openAndAssertVisible() {
-    await this.page.keyboard.press('Control+KeyF');
+  async openAndAssertVisible(browserName: string) {
+    await this.page.keyboard.press(cmdCtrl(browserName) + '+F');
```

Comment:
'ControlOrMeta+KeyF' was not working?
this should automatically use meta on webkit and control on windows/linux

Reply:
then you don't need to use the browserName

Reply:
it works on my machine but not on jenkins

Reply:
Also saw once that this not worked, just wanted to mention this if you don't know and you didn't tried it.